en/micro-integrator/docs/setup/deployment/databases/setting-up-IBM-DB2.md

```diff
@@ -65,4 +84,16 @@ pool_options.maxWait = 60000
 pool_options.testOnBorrow = true
 ```
 
+```toml tab='User DB Connection'
+[[datasource]]
+id = "WSO2_USER_DB"
```

Comment:
We need to configure the realm to use this datasource
```
[realm_manager]
data_source = "WSO2_USER_DB"
```

Reply:
same with all the db types.

Reply:
fixed